TheWitcher/Menu.cs

```diff
@@ -0,0 +1,152 @@
+                        return;
+                }
+            } while (GameIsActive);
+        }
+        private void SelectStore()
+        {
+            Console.WriteLine("Для выбора магазина оружия - нажмите 1");
+            Console.WriteLine("Для выбора магазина одежды - нажмите 2");
+            Console.WriteLine("Для выхода в главное меню  - нажмите 3");
+            string storeSelected = Console.ReadLine();
+            do
+            {
+                if (int.TryParse(storeSelected, out var num) && num > 0 && num < 4)
+                {
+                    if (num == 1)
+                    {
+                        DisplayWeaponsStore();
+                        return;
+                    }
+                    else if (num == 2)
+                    {
+                        DisplayClothesStore();
+                        return;
+                    }
+                    else
+                    {
+                        return;
+                    }
+                }
+            }
+            while (true);
+        }
+        private void DisplayWeaponsStore()
+        {
+            Console.WriteLine("Вы находитесь в магазине оружия, для выбора, укажите соответствующий номер");
+        }
+        private void DisplayClothesStore()
+        {
+            Console.WriteLine("Вы находитесь в магазине одежды, для выбора, укажите соответствующий номер");
+        }
+        private void GetHeroStatistic(HeroesDTO heroDTO)
+        {
+            Console.WriteLine("Имя :" + heroDTO.HeroName);
+            Console.WriteLine("Уровень :" + heroDTO.HeroLevel);
+            Console.WriteLine("Описание :" + heroDTO.HeroDescription);
+            Console.WriteLine("Деньги :" + heroDTO.HeroMoney);
+        }
+        private void DisplayAvailableQuests(HeroesDTO hero)
+        {
+            List<QuestDTO> questsDTO = _questService.GetNameIdLeadTimeQuests(hero);
+            foreach(var quest in questsDTO)
+            {
+                Console.WriteLine("{0} - {1}, квест займёт {2}, сложность квеста - {3}", quest.Id, quest.QuestName, quest.LeadTime, quest.Complexity);
+            }            
+        }
+        private void ChooseQuest(int heroId)
+        {
+            Console.WriteLine("Выберете квест, указав соответствующую цифру, сложность квеста напрямую влияют на время выполнения и награду за квест");
+            var questSelected = Console.ReadLine();
+            int questId = Convert.ToInt32(questSelected);
+            bool questIsTaken = _heroService.TakeTheQuest(heroId, questId);
+            string questIsTakenConsoleOutput;
+            if (questIsTaken)
+            {
+                questIsTakenConsoleOutput = "Квест успешно начат";
+            }
+            else
+            {
+                questIsTakenConsoleOutput = "Квест не доступен";
+            }
+            Console.WriteLine(questIsTakenConsoleOutput);
+        }
+
+        private static UserSelection SelectGame()
```

Comment:
Are you able to build this? There seems to be no return statement (except of inside if) so not all code parts return value